Microsoft.Azure.Cosmos/src/Resource/FullFidelity/ChangeFeedItem{T}.cs

```diff
@@ -0,0 +1,37 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace Microsoft.Azure.Cosmos.FullFidelity
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// The resource of full fidelity change feeds.
```

Comment:
We renamed Full Fidelity to All Operations but the comments still say Full Fidelity. Maybe instead we can say something like:

```suggestion
    /// Response that contains the metadata information when using <see cref="ChangeFeedMode.AllOperations" /> to consume the change feed
```